scripts/reviewers/teams.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "name": "core",
+    "users": [
+      "Udumft",
+      "chezzdev",
+      "kried"
+    ]
+  },
+  {
+    "name": "ui",
+    "users": [
```

Comment:
I wanted to suggest using Slack API [usergroups](https://api.slack.com/methods/usergroups.users.list) to get users and avoid data duplications here. But unfortunately, it looks like we only have this Core/UI/Apps separation described in confluence.

Could you please then update [the onboarding guide](https://mapbox.atlassian.net/wiki/spaces/NAVSDK/pages/51314696/iOS+Engineer+Onboarding+Guide#Getting-to-know-the-team) to not forget to add new members to the github teams?

Reply:
Done